claudette/core.py

```diff
@@ -57,6 +58,11 @@
 # %% ../00_core.ipynb
 text_only_models = ('claude-3-5-haiku-20241022',)
 
+# %% ../00_core.ipynb
+has_streaming_models = set(all_models)
+has_system_prompt_models = set(all_models)
+has_temperature_models = set(all_models)
```

Comment:
It might be worth adding a note to say that these properties exist because we wanted to provide a unified interface so that people can easily swap between Claudette and Cosette?